src/main/scala/org/apache/spark/sql/kinesis/KinesisSourceOffset.scala

```diff
@@ -79,5 +85,24 @@ object KinesisSourceOffset {
     }
   }
 
+  /*
+   * Returns [[KinesisSourceOffset]] from a JSON
+   */
+  def apply(json: String): KinesisSourceOffset = {
```

Comment:
Since this function is similar to the one above, can we call apply(so.json) in the above function to avoid duplication?